commcare_connect/opportunity/views.py

```diff
@@ -1524,6 +1524,26 @@ def breadcrumb_title(self):
         return _("Review Custom Invoice")
 
 
+@org_member_required
+@opportunity_required
+@require_POST
+def submit_invoice(request, org_slug, opp_id):
+    if not (request.opportunity.managed and request.org_membership):
+        return redirect("opportunity:detail", org_slug, opp_id)
+
+    invoice_id = request.POST.get("invoice_id")
+    invoice = get_object_or_404(PaymentInvoice, opportunity=request.opportunity, pk=invoice_id)
+    if invoice.status != InvoiceStatus.PENDING:
+        return HttpResponseBadRequest("Only invoices with status 'Pending' can be submitted for approval.")
+
+    invoice.status = InvoiceStatus.SUBMITTED
+    invoice.save()
+    messages.success(request, _("Invoice %(invoice_number)s submitted for approval.") % {
```

Comment:
Nit: Is there a reason for the "s" here?
```suggestion
    messages.success(request, _("Invoice %(invoice_number) submitted for approval.") % {
```